build/ci-build.yml

```diff
@@ -14,10 +14,21 @@ pr:
     - src/*
     - build/ci-build.yml
 
+resources:
```

Comment:
We'll need to remove this

Reply:
will update to reference the templates repo